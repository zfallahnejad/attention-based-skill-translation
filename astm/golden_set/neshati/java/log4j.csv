870248,log4j
592139,log4j
21234,log4j
605744,log4j
57695,log4j
