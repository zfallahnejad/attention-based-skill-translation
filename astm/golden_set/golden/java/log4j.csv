870248,32,40,0.4444444444444444
592139,20,10,0.6666666666666666
21234,14,19,0.42424242424242425
605744,12,8,0.6
57695,11,13,0.4583333333333333
22656,10,9,0.5263157894736842
265143,10,12,0.45454545454545453
443716,10,8,0.5555555555555556
